omod/src/main/webapp/uiError.jsp

```diff
@@ -1,12 +1,7 @@
-<%@ include file="/WEB-INF/template/include.jsp"%>
-<%@ include file="/WEB-INF/template/header.jsp"%>
-
 <h1>UI Framework Error</h1>
```

Comment:
Should we only show the error page when there is an authenticate user who elevated privileges?